docs/DeveloperGuide.adoc

```diff
@@ -154,30 +152,36 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 This section describes some noteworthy details on how certain features are implemented.
 
 // tag::add[]
-=== Add Eatery feature
+=== Add Eatery feature : `add`
 ==== Implementation
 
-The add mechanism is facilitated by `AddressBook`. It implements the following operations:
+The add feature allows the user to add an eatery with the compulsory fields: `Name`, `Address`, and `Category`. The `Tag` field(s), however, is an optional field. The user can add tag(s) to the eatery later on using the `addtag` feature, if needed. The add mechanism is facilitated by `EateryList`. It implements the following operations:
 
-* `AddressBook#hasEatery()` -- Checks if an Eatery with the same identity as Eatery e exists in the address book.
-* `AddressBook#addEatery()` -- Adds an Eatery to the address book. The eatery must not already exist in the address book.
+* `EateryList#hasEatery(Eatery eatery)` -- Checks if an Eatery with the same identity as Eatery eatery exists in the address book.
+* `EateryList#addEatery(Eatery eatery)` -- Adds an Eatery eatery to the address book. The eatery must not already exist in the address book.
```

Comment:
i think can leave out the parameters. Also, should we rename the addressbook?

Reply:
omg ya i forgot to rename thank u joseph